---
layout: page
title: CV
permalink: /cv/
---

Phillip T. Brooks, Ph.D.
========================

--------------------------------
(517) 944-8954 
325 K street #91 Davis, CA 95616
ptbrooks@ucdavis.edu github.com/brooksph linkedin.com/in/philliptbrooks
--------------------------------

## SUMMARY OF QUALIFICATIONS

Phillip is a co-chair for the NIH DCPPC Interoperability Task Force, Certified Data Carpentry Instructor, former Associate Director of Data Intensive Biology Institute at the University of California, Davis, and member of the American Association for Mi- crobiology (ASM), with years of experience in microbiology, immunology, microbiome analysis, open source project development and cloud computing. Phillip is an experi- enced developer of genomic and metagenomic analysis pipelines, responsible for project management and data generation, interpretation, and presentation.

## EDUCATION

Michigan State University, East Lansing, MI
Doctor of Philosophy, Dual-degree Comparative Medicine and Integrative Biology (CMIB) and Environmental Toxicology, August 2016

Georgia State University, Atlanta, GA
Bachelor of Science, Psychology, Chemistry minor, December 2010

## STRENGTHS & SKILLS

**Professional Certifications**: Data Carpentry
**Languages**: Bash scripting, some Python, some R
**Python Libraries**: numpy, pandas, seaborn, matplotlib, and Jupyter
**Workflow Management and Automation**: Snakemake, Docker, and Singularity Cloud and Big Data: Amazon Web Services (AWS), Jetstream, SLURM, and TORQUE
**Project Management and Version Control**: Git/Github and the Open Science Framework (osf.io)
**High Throughput Sequence Analysis**: Read quality score assessment and filter- ing (fastqc, trimmomatic), genome and metagenome assembly (SPAdes, MEGAHIT), genome alignment, mapping and variant calling (BWA and bcftools), taxonomic clas- sification (sourmash and kaiju), functional annotation (prokka, SRST2, ABRicate, and mi-faser), and amplicon analysis (mothur)
**Laboratory skills**: Basic microbiology techniques (media preparation, cell culture, basic and differential staining, etc); cell culture, sonication, ammonium sulfate precipi- tation, centrifugation, fast protein liquid chromatography (FPLC), gel electrophoresis, enzyme linked immunosorbent assay (ELISA), stopped-flow spectrophotometry, flow cytometry, western blotting, gel electrophoresis, RNA/DNA/protein isolation, PCR, quantitative real-time PCR, immunocytochemistry, immunohistochemistry, light and fluorescent microscopy, mouse breeding, generation of humanized microbiota mice from germ-free mice, housing and genotyping; open-field test, hang test; cell viability assessment

## EXPERIENCE

**Postdoctoral Scholar Dept. of Population Health and Repro.** 
2016–Present University of California, Davis
  - Project management of 4-person team, building metagenomics workflows (https://github.com/dahak-metagenomics/dahak)
  - Design, execution, and interpretation of scientific results from metagenome work- flow system
  - Coordinating successful interdisciplinary collaborations in a team setting
  - Developing open-source software for shotgun metagenome analysis and meets or exceeds all contractual milestones
  - Installing, maintaining, and deploying bioinformatics tools in Docker containers
  - Developing and deploying workflows automated with Snakemake on cloud com- puters, (AWS, Jetstream) and high performance compute clusters
  - Teaching local, national, and international workshops (https://github.com/dib- lab/2017-dibsi-metagenomics, https://github.com/ngs-docs/2017-cicese-metagenomics)

**Co-chair NIH DCPPC Dept. of Population Health and Repro. Interoperability Task Force**
2018-Present University of California, Davis
  - Ensures interoperability between unique full stack implementations of the NIH data commons
  
**Associate Director Dept. of Population Health and Repro.**
2016-2017 University of California, Davis
  - Planned and coordinated a 5-week Data Intensive Biology Summer Institute at UC Davis with 150 participants from 5 continents
  - Developed and taught course materials on metagenomics and genomics to more than 200 students

**Graduate Fellow Dept. of Microbiology and Mol. Gen.**
2011-2016 East Lansing, Michigan
  - Demonstrated excellent analytical, organizational and multitasking skills
  - Trained undergraduates and graduate students
  - Designed and executed high throughput amplicon sequencing microbiome exper- iments using mothur, R, and Python
  - Compared and analyzed single nucleotide variants in resequenced bacterial genomes
  - Tested the hypothesis that the composition of gut microbiota is a large factor in the manifestation of autoimmunity in a Campylobacter jejuni infected murine model
  - Discovered limited diversity and Lactobacillus deficient microbiota increased host susceptibility to C. jejuni initiated autoimmunity and was documented in a one published manuscript and one manuscript under review

**Graduate Fellow Dept. of Pathology and Diagnostic Investigation**
2011 East Lansing, Michigan 
- Investigated the interaction of air pollution and high-fructose diet using a murine
model of fructose-induced cardio-metabolic syndrome. Our results revealed a synergistic effect of pollution and high fructose diet that promoted allergic airway responses. These data contributed to the publication of a manuscript

**Ronald E. McNair Program Michigan State University Undergraduate Visiting Scholar**
2009 East Lansing, Michigan 
  - Investigated the fatty acid palmitate as an inhibitor of ATP binding protein
kinase r (PKR), protein kinase b (PKB), and adenosine monophosphate-activated protein kinase (AMPK)
  - Developed methods for the purification of protein kinase r

**Howard Hughes Biotechnology Michigan State University Scholars program**
2009 East Lansing, Michigan 
  - Conducted the first ever purification and characterization of D-Arginine dehydrogenase
  - Results obtained contributed to a manuscript

**Ronald E. McNair Program Michigan State University Undergraduate Visiting Scholar**
2008 East Lansing, Michigan
  - Probed the mechanism of Zn2+ activation of c-peptide, a component of pro-insulin
  - Result demonstrated that Zn2+ increased glucose uptake by red blood cells

## RESEARCH GRANTS

Co-authored proposal to Joint Genome Institute (JGI). Awarded access to data and computational resources. With C. Titus Brown titled ”Advancing metagenome classi- fication and comparison by MinHash fingerprinting of IMG/M data sets”, March 2017

## HONORS

Institute for Integrative Toxicology Fellowship, 2012-2015
Alliance for Graduate Education Fellowship, Spring 2014 and 2015
Best Oral Presentation by a Ph.D. Candidate at Phi Zeta Day, Fall 2014
Society of Toxicology Annual Meeting (Baltimore, Maryland) Student Travel Award Winner, 2009
Michigan State University Ronald E. McNair Spirit Award, 2008

## PROFESSIONAL AFFILIATIONS
  
Member, American Society for Microbiology (ASM) 

Member, Society of Toxicology (SOT)
 
## PUBLICATIONS

**IN PRESS**

Phillip T. Brooks, Linda S. Mansfield (2018). Effects of antibiotic resistance (AR) and microbiota shifts on Campylobacter jejuni-mediated disease. Animal Health Research Reviews.

**PUBLISHED**

Phillip T. Brooks, Kelsey A. Brakel, Julia A. Bell, Christopher E. Bejcek, Trey Gilpin, Jean M. Brudvig, and Linda S. Mansfield (2017). Transplanted human fecal mi- crobiota enhanced Guillain Barr syndrome autoantibody responses after Campylobacter jejuni infection in C57BL/6 mice. BMC Microbiome DOI:10.1186/s40168-017-0284-4

Chee Bing Ong, Kazuyoshi Kumagai, Phillip T. Brooks, Christina Brandenberger, Ryan P. Lewandowski, Daven N. Jackson-Humbles, Rance Nault, Timothy Zacharewski, James G. Wagner, and Jack R. Harkema (2015). Ozone-Induced Type 2 Immunity in Nasal Airways: Development and Lymphoid Cell Dependence in Mice. American Jour- nal of Respiratory Cell and Molecular Biology. DOI:10.1165/rcmb.2015-0165O

Hongling Yuan, Guoxing Fu, Phillip T. Brooks, Irene Weber, & Giovanni Gadda. (2010). Steady State Kinetic Mechanism and Reductive Half-Reaction of D-Arginine dehydrogenase from Pseudomonas aeruginosa. Journal of Biochemistry, 49, pp. 9542- 9550 DOI: 10.1021/BI101420w

**UNDER REVIEW**

Phillip T. Brooks, Julia A. Bell, Christopher E. Bejcek, Ankit Malik, and Linda S. Mansfield. Antibiotic depletion drives severe Campylobacter jejuni-mediated Type 1/17 colitis and Type 2 autoimmunity

## ABSTRACTS AND PRESENTATIONS

**ORAL**

Phillip T. Brooks (2017). Composition of Gut Microbiota Affects Campylobacter jejuni Mediated Inflammation and Autoimmunity in Murine Models. microBOOmes: UC Davis Genome Center Halloween Symposium on Microbiomes, University of California, Davis, Davis, CA

Phillip T. Brooks (2016). Composition of Gut Microbiota Affects Campylobacter jejuni Mediated Inflammation and Autoimmunity in Murine Models. Ph.D. Defense seminar, Michigan State University, East Lansing, MI

Phillip T. Brooks (2015). Host Microbiota as a Factor in the Adaptation of Campy- lobacter jejuni Guillain-Barr Associated Strains. Proposal Seminar, Michigan State University, East Lansing, MI

Phillip T. Brooks (2014). Characterization of Campylobacter jejuni Infection in C57BL/6 IL-10-/- Mice with Conventional or Antibiotic Altered Microbiota . MSU College of Veterinary Medicine Annual Phi-Zeta day, Michigan State University, East Lansing, MI

Phillip T. Brooks (2014). Pathogens, antibiotics and evolution. MSU Alliance for Graduate Education and the Professoriate (AGEP), Michigan State University, East Lansing, MI

Phillip T. Brooks (2009). Biophysical Interaction of palmitate with the kinase do- main of Protein Kinase R (PKR). McNair/SRI Symposium, Michigan State University, East Lansing, MI

Phillip T. Brooks (2008). Metal (Zn2+) Activated C-peptide Induced ATP Release via Red Blood Cells (RBCs) with the Addition of Albumin. Annual Biomedical Re- search Conference for Minority Students in Orlando, FL

Phillip T. Brooks (2008). Metal (Zn2+) Activated C-peptide Induced ATP Release via Red Blood Cells (RBCs) with the Addition of Albumin. McNair/SROP Sympo- sium, Michigan State University, East Lansing, MI

**POSTER**

Phillip T. Brooks, Bruce Budowle, Chris Grahlmann, Stephanie L. Guertin, F. Curtis Hewitt, Dana R. Kadavy, Alexander F. Koeppel, Oana I. Lungu, Krista L. Ternus, Stephen D. Turner, and C. Titus Brown (2017). A Sustainable Approach to High Confidence Metagenomics Analysis of Complex Samples. 2nd ASM Conference on Rapid Next-generation Sequencing and Bioinformatic Pipelines, Washington, DC

Phillip T. Brooks, Kelsey Brakel, Alexander Etheridge, Julia Bell, and Linda Mans- field (2017). A Human-derived Microbiome Enhanced Campylobacter jejuni Coloniza- tion Levels in C57BL/6 and NOD Mice in Association with Likely Changes in the Gut Microbial Food Web. Gordon Research Conference

Linda S. Mansfield, Phillip T. Brooks, Kelsey A. Brakel, Julia A. Bell, Susan L. Ewart, Daniel Claiborne, and Syed H. Arshad (2017). Developing transplanted human microbiota mouse models to study the effects of the early microbiome on development of inflammation, autoimmunity and allergy. 6th Annual in-FLAME (Inflammation, Ecosystems, and the transformation of Global Health) Workshop, New York, NY

Phillip T. Brooks, Luiz C. Irber Jr., Taylor E. Reiter, and C. Titus Brown (2017). Taxonomic Classification Of Microbial Metagenomes Using MinHash Signatures. ASM Microbe, New Orleans, LA

Phillip T. Brooks, Julia A. Bell, Christopher E. Bejcek, Ankit Malik, Linda S. Mans- field (2016). Broad Spectrum Antibiotic Treatment Enhanced T Cell Mediated Inflam- mation and Guillain Barr-Associated Antibody Responses to Campylobacter jejuni in a Mouse Model. Society for Toxicology Annual Meeting, New Orleans, LA

Phillip T. Brooks, Kelsey A. Brakel, Julia A. Bell, Linda S. Mansfield (2015). Hu- manized microbiota alter host immune response to Campylobacter jejuni infection. Society for Neuroscience Annual Meeting, Chicago, IL

Phillip T. Brooks, Ankit Malik, Julia A. Bell, Linda S. Mansfield (2013). 

## TEACHING EXPERINECE

Characterization of Campylobacter jejuni Infection in C57BL/6 IL-10-/- Mice with Conventional or Antibiotic Altered Microbiota. Society for Microbiology Annual Meeting, Denver, CO

Phillip T. Brooks and Christina Chan (2009). Biophysical Interaction of palmi- tate with the kinase domain of Protein Kinase R (PKR). McNair/SROP Symposium, Michigan State University, East Lansing, MI

Phillip T. Brooks and Dana Spence (2008). Metal (Zn2+) Activated C-peptide Induced ATP Release via Red Blood Cells (RBCs) with the Addition of Albumin . Ronald E. McNair Symposium, Georgia State University, Atlanta, GA

Phillip T. Brooks and Dana Spence (2008). Metal (Zn2+) Activated C-peptide Induced ATP Release via Red Blood Cells (RBCs) with the Addition of Albumin . CIC/SROP Conference, Michigan State University, East Lansing, MI

Phillip T. Brooks and Dana Spence (2008). Metal (Zn2+) Activated C-peptide Induced ATP Release via Red Blood Cells (RBCs) with the Addition of Albumin . McNair/SROP Symposium, Michigan State University, East Lansing, MI

2017 UCD Data Intensive Biology Summer Institute ANGUS 2-Week workshop, Uni- versity of California, Davis Associate director and instructor
  - http://angus.readthedocs.io/en/2017/, http://ivory.idyll.org/dibsi/
  
2017 UCSC Metagenomics workshop, University of California Santa Cruz, Co-instructor,
  - https://2017-ucsc-metagenomics.readthedocs.io/en/latest/
  
2017 University of California, Davis Data Intensive Biology Summer Institute Environmental Metagenomics Workshop 2015 Beacon Next Generation Sequencing Course, Michigan State University, Teaching Assistant
  - http://angus.readthedocs.org/en/2015/

## CIVIC AND COMMUNITY ENGAGEMENT

Diversity Committee, College of Veterinary Medicine, Co-chair, 2015-2016

Strategic Planning Diversity Committee, College of Veterinary Medicine, CORE team member, 2015

Graduate Facilitator, Michigan State University Summer Research Opportunities Pro- gram (SROP), 2014-2016

Graduate Facilitator, Michigan State University Bridge to the Ph.D. in Neuroscience Program (BPNP), 2012-2014

Graduate Facilitator, Michigan State University ENDURE Summer Program, 2015

